README.md

```diff
@@ -154,6 +154,21 @@ $object = $bucket->object('file_backup.txt');
 $object->downloadToFile('/data/file_backup.txt');
 ```
 
+#### Stream Wrapper
+
+```php
+require 'vendor/autoload.php';
+
+use Google\Cloud\Storage\StorageClient;
+
+$storage = new StorageClient([
+    'projectId' => 'my_project'
```

Comment:
Our inline documentation (code snippets in docblocks) generally do not provide a project ID. I'd like to update the README to conform to this as well at some point, but I agree with you that more detail in general regarding different means of authentication is in order.